service/lib/agama/manager.rb

```diff
@@ -159,6 +148,21 @@ def install_phase
     end
     # rubocop:enable Metrics/AbcSize
 
+    def locale=(locale)
```

Comment:
Thanks for moving this. I always think that it is misplaced, but I never fix it.